.github/workflows/continuous.yml

```diff
@@ -108,9 +110,9 @@ jobs:
 
     - name: Run rack-box tests
       run: |
-        cd RACK
-        pip3 install -r cli/requirements.txt
-        pip3 install cli/.
+        cd RACK/cli
+        pip3 install .
+        cd ..
```

Comment:
Too bad that you need two cd commands.  What makes `pip3 install ./cli` no longer work?

Reply:
First I tried what you wrote, and that didn't work. I'm new to using `pyproject.toml` and when I found that the code committed above working, I went with it.

Your question prompted me to *try harder* and I learned that a trailing `/` indicates to pip that what I want is for it to install a directory, rather than a package name. I've updated the script to use a single command and saved the extra `cd` for another day!